README.md

```diff
@@ -55,30 +48,52 @@ Follow these steps to setup and run this Code Pattern. The steps are
 described in detail below.
 
 1. [Sign up for the Watson Studio](#1-sign-up-for-watson-studio)
-2. [Create the notebook](#2-create-the-notebook)
-3. [Run the notebook](#3-run-the-notebook)
-4. [Analyze the results](#4-analyze-the-results)
-5. [Save and Share](#5-save-and-share)
+2. [Create the Spark Service](#2-create-the-spark-service)
+3. [Create the notebook](#3-create-the-notebook)
+4. [Run the notebook](#4-run-the-notebook)
+5. [Analyze the results](#5-analyze-the-results)
+6. [Save and Share](#6-save-and-share)
 
 ## 1. Sign up for Watson Studio
 
 Sign up for IBM's [Watson Studio](https://dataplatform.cloud.ibm.com/). By creating a project in Watson Studio a free tier ``Object Storage`` service will be created in your IBM Cloud account. Take note of your service names as you will need to select them in the following steps.
 
 > Note: When creating your Object Storage service, select the ``Free`` storage type in order to avoid having to pay an upgrade fee.
 
-## 2. Create the notebook
+## 2. Create the Spark service
```

Comment:
this is out of date

Reply:
Yes, I have this and other things to fix. this is just an initial commit.